src/main/scala/org/youshuren/persistence/queries/BookQueries.scala

```diff
@@ -0,0 +1,20 @@
+package org.youshuren.persistence.queries
+
+import doobie.postgres.implicits._
+import doobie.implicits._
+import doobie.util.query.Query0
+import org.youshuren.model.WeChatUser
+
+object BookQueries {
```

Comment:
How do you like that?

Reply:
I've put similar Query & Update objects in Postgres object since it's coupled to implementation. in this way we encapsulate Postgres implementation in a trait and just mix in it into the Database[A] type class. In any chance if we'd like to switch to other lib like Redis, MongoDB etc. we can just replace or mix in them